windows/NsisInclude/FilesToUninstall.nsh

```diff
@@ -323,7 +323,6 @@ Delete /REBOOTOK "$INSTDIR\libwebp-5.dll"
 Delete /REBOOTOK "$INSTDIR\libwinpthread-1.dll"
 Delete /REBOOTOK "$INSTDIR\libxmlxpat.dll"
 Delete /REBOOTOK "$INSTDIR\libzzz.dll"
-Delete /REBOOTOK "$INSTDIR\mfc100u.dll"
```

Comment:
@rados 
Do you now why this was removed? If not, no need to investigate

Reply:
I don't. Actually those files  - FilesToUninstall.nsh and FilesToInstall.nsh are generated every time when we run the build procedure and probably we don't need to keep them in the repository. Back then I put them in just for reference and as a safe option because the fix was a recent one. Now I don't see a reason to keep them any longer in the repository...

Reply:
Sometimes it is good to track autogenerated files in the repo for troubleshooting generators.